assets/src/scss/components/_buttons.scss

```diff
@@ -12,23 +12,25 @@
 .btn,
 .wp-block-button a,
 .wp-block-file .wp-block-file__button {
+  --button-- {
+    font-family: $roboto;
+    text-align: center;
+    font-weight: bold;
+    border-radius: 4px;
+    padding: 0 30px;
+    transition-property: color, background-color, border-color;
+    transition-duration: 150ms;
+    transition-timing-function: linear;
+  }
   display: inline-block;
   position: relative;
-  font-family: $roboto;
   font-size: $font-size-sm;
-  text-align: center;
   text-decoration: none;
   color: $white;
```

Comment:
Also this color maybe? (and later in the file, L32, the line-height?)

Reply:
This one I didn't make variable because all other classes override it. Maybe we can remove this declaration too, but probably best separately.

Reply:
There is actually one use case where I think it's used, it's for the Button block when users manually select the white text color... But indeed it doesn't need to be investigated in this PR 🙂